### ISSUE TITLE
WebView displaced on Android when built with version > 4.0.3

### ISSUE THREAD

**Issue by cvuorinen** [AppGyver/steroids#909]
I have an app that is a pretty basic Ionic app. It was started before supersonic so not using it.
When I build it with version 4.0.4 or 4.0.5, there's an empty grey bar on top of the header bar.

Inspecting with Chrome remote inspector shows that the whole WebView seems to be displaced as it only starts below the grey bar.

Here is a screenshot when inspecting it. The light grey area is the problem. When built with 4.0.3 the darker grey bar with the title is right below the black Android status bar as it should.
![ccshgrhwiaa2soq](https://cloud.githubusercontent.com/assets/1764167/7189226/1e3d2c5e-e486-11e4-8f7b-6f6513dad453.jpg)

**Comment by harsaharsa** [comment]
Are you using the Crosswalk or Platform WebView? Does it happen on all devices? What Android version? (Trying to figure out the conditions – haven't been able to replicate this on our end.)

**Comment by cvuorinen** [comment]
OK, seems like the grey area is actually the native navigation bar feature that was added in supersonic. I copied the `native-styles` directory from a new project I generated and it changed to blue.

As I said, this project is pre supersonic so I'm currently not using it. So is there some config option to disable the native navigation bar feature?

**Comment by cvuorinen** [comment]
I'm using Platform WebView but it happens with the scanner app also. I have Android 5.0 on a Nexus 5.

**Comment by harsaharsa** [comment]
Alright, that explains it then. The navigation bar is not visible by default – you should either have a `<super-navbar>` element or a call to `supersonic.ui.navigationBar.show()` somewhere that's causing this.

**Comment by cvuorinen** [comment]
Nope, don't have either of those. I don't even have supersonic as dependency, just steroids-js.

**Comment by harsaharsa** [comment]
That sounds very strange. If you just load an empty HTML file with no content, does it show the nav bar still?

**Comment by cvuorinen** [comment]
Yep, still there. Otherwise just a blank white page but the grey bar is still there.
So it must be something about my configs?

My project still has the old style application.coffee with `steroids.config.name = "app-name"` etc. instead of the new way with app.coffee and structure.coffee that's used in the new project that I created to compare with. Should the old config still work like before or should I try to update it?

**Comment by harsaharsa** [comment]
The configs shouldn't initiate the navigation bar... Old Steroids configs should work fine. Does it happen with a new project?

**Comment by cvuorinen** [comment]
I created a new project with `steroids create` and then removed the `super-navbar` element from index.html. Otherwise left everything as is.

Scanner: Show the nav bar still.

Originally it was blue because of the native-styles were still there. After I commented out the `navigation-bar` styles, it changed to white.

Then I deployed and built with few different versions.

4.0.4 (Crosswalk, ARM only): Shows the nav bar.

4.0.5-edge1 (Platform WebView): Shows the nav bar.

4.0.5-edge12 (Platform WebView): Does not show the nav bar!

So some version between 4.0.5-edge1 and edge12 has fixed it.
